client/devdocs/sidebar.jsx

```diff
@@ -87,7 +87,15 @@ export default class DevdocsSidebar extends PureComponent {
 						link="/devdocs/docs/i18n.md"
 						selected={ this.isItemSelected( '/devdocs/docs/i18n.md' ) }
 					/>
+					<SidebarItem
+						className="devdocs__navigation-item"
+						icon="bug"
+						label="Testing"
+						link="/devdocs/docs/testing/index.md"
+						selected={ this.isItemSelected( '/devdocs/docs/testing', false ) }
+					/>
 
+					<br></br>
```

Comment:
I believe we're supposed to avoid using `br` for style and layout purposes. What do you think instead of adding a class to the headings that need extra space above them and adding `margin` in the [devdocs styles](https://github.com/Automattic/wp-calypso/blob/trunk/client/devdocs/style.scss)?

Reply:
I fixed this by using `SidebarSeparator` class and then adding a 20px padding on the class in devdocs stylesheet.